src/main/resources/view/HelpWindow.fxml

```diff
@@ -14,6 +15,10 @@
   </icons>
   <scene>
     <Scene>
+        <stylesheets>
+            <URL value="@Fonts.css"/>
+            <URL value="@HelpWindow.css"/>
```

Comment:
Not something directly related to fonts, more of a general comment for the project, why isn't the dark theme applied to the help window? If it uses the same css means it will be more consistent as well.

Reply:
Good point @kouyk . Let's tackle it separately. Create an issue for it?